Scrum-97-things-guide.md

```
@@ -0,0 +1,639 @@
+7. [Part VI. 熟達こそが重要（Mastery Does Matter）](#6)
+8. [Part VII. 人はどこまでも人間である（People, All Too Human）](#7)
+9. [Part VIII. 価値基準が行動を導く（Values Drive Behavior）](#8)
+10. [Part IX. 組織デザイン（Organizational Design）](#9)
+11. [Part X. 台本のないスクラム（Scrum Off Script）](#10)
+12. [まとめ：実践チェックリスト](#11)
+13. [参考文献・出典URL一覧](#12)
+
+---
+
+<a id="0"></a>
+## 0. このガイドについて
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)